app/views/pages/home.js.erb

```diff
@@ -0,0 +1,4 @@
+$(function() {
+  <%# Only update the button; no need to update anything else %>
+  $(".<%= feed_item_action_button_container_css_class(@feed_item.id) %>").html("<%= j (render 'shared/recommended_button', feed_item: @feed_item) %>")
+})
```

Comment:
Thrifty. When a user recommends (stars) a feed item, we're ONLY updating the item in the UI that has been recommended - better user experience. Likewise, we are only pulling from the database this single feed item on the back end - efficient.